demo/kubernetes/monitoring/prometheus.yaml

```diff
@@ -1,9 +1,52 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+  namespace: rook
+---
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRole
+metadata:
+  name: prometheus
+  namespace: rook
+rules:
+- apiGroups: [""]
+  resources:
```

Comment:
is that a question about syntax in these files, or do you already know that would work and just would like to see it that way to clean up a bit?

I got this example from the prometheus-operator repo, so I would probably like to keep it as they had it to make it easier to identify and deltas if they make changes in the future...